packages/atlas/src/components/_nft/NftWidget/NftHistory.tsx

```diff
@@ -0,0 +1,178 @@
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Bid made by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Listed by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      copy: 'Auction settled by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+  ]
+
+  return (
+    <>
+      <NftHistoryHeader data-open={isOpen} data-size={size} onClick={toggleIsOpen}>
+        <Text variant={size === 'small' ? 'h300' : 'h400'}>History</Text>
+        <StyledChevronButton data-open={isOpen} iconOnly variant="tertiary" icon={<SvgActionChevronB />} />
+      </NftHistoryHeader>
+      {isOpen && (
+        <div style={{ position: 'relative' }}>
+          <FadingBlock data-size={size} width={width} />
+          <HistoryPanel width={width} data-size={size} data-open={isOpen}>
+            {/* TODO: remove dummy data */}
+            {dummyData.map((props: HistoryItemProps, index) => (
+              <HistoryItem key={index} {...props} />
+            ))}
+          </HistoryPanel>
+          <FadingBlock data-size={size} width={width} data-bottom />
+        </div>
+      )}
+    </>
+  )
+}
+
+type HistoryItemProps = {
+  size: Size
+  memberAvatarUri: string
+  date: Date
+  joyAmount?: number
+  dollarValue?: string
+  copy: string
+  memberHandle: string
+}
+export const HistoryItem: React.FC<HistoryItemProps> = ({
+  size,
+  memberAvatarUri,
+  date,
+  joyAmount,
+  dollarValue,
+  copy,
+  memberHandle,
+}) => {
+  const navigate = useNavigate()
+  return (
+    <HistoryItemContainer data-size={size}>
+      <Avatar
+        onClick={() => navigate(absoluteRoutes.viewer.member(memberHandle))}
+        assetUrl={memberAvatarUri}
+        size={size === 'medium' ? 'small' : 'default'}
+      />
+      <TextContainer>
+        <CopyContainer>
+          <Text variant={size === 'medium' ? 'h300' : 'h200'} secondary>
+            {copy}{' '}
+            <OwnerHandle to={absoluteRoutes.viewer.member(memberHandle)} variant="secondary" textOnly>
+              <Text variant={size === 'medium' ? 'h300' : 'h200'}>{memberHandle}</Text>
+            </OwnerHandle>
+          </Text>
+        </CopyContainer>
+        <Text variant="t100" secondary>
+          {format(date, "d MMM yyyy 'at' HH:mm")}
```

Comment:
Date formatting is a little bid different in figma
![Zrzut ekranu 2022-03-9 o 20 05 46](https://user-images.githubusercontent.com/13706246/157513516-42cbd622-c53f-4c31-8d13-e899ba9eb499.png)